paddlets/datasets/tsdataset.py

```diff
@@ -290,7 +290,7 @@ def to_numpy(self, copy: bool=True) -> np.ndarray:
     def get_index_at_point(
         self, 
         point: Union[pd.Timestamp, str, float, int], 
-        after=True
+        after: Optional[bool]=True
```

Comment:
Line294 after should Not be Optional, refers to https://docs.python.org/3/library/typing.html#typing.Optional